react-java-mysql/backend/Dockerfile

```diff
@@ -1,10 +1,30 @@
+# syntax=docker/dockerfile:1.4
+
 FROM --platform=$BUILDPLATFORM maven:3.8.5-eclipse-temurin-17 AS builder
 WORKDIR /workdir/server
 COPY pom.xml /workdir/server/pom.xml
 RUN mvn dependency:go-offline
 
 COPY src /workdir/server/src
 RUN mvn install
+
+FROM builder as dev-envs
+
+RUN <<EOF
+apt-get update
+apt-get install -y git
+EOF
+
+RUN <<EOF  \
```

Comment:
nit: we probably don‘t need the backslash here

Reply:
My IDE think it smarter than me but it don't know anything about Heredoc notation 🤪